### PR TITLE
Convert test script for segmentizing to doc plot

### PR DESCRIPTION
**Pull request by PaulWessel**
See #7911 for background.  Since **plot -F** is a bit obscure it can really benefit from an illustration.  We had two tests for this and I have removed one of them and converted the other to a tighter and better legend figure for `plot.rst`.  Figure looks like this (based on old _test/psxy/geosegmentize.sh_ that is now deleted, replaced by _doc/scripts/GMT_segmentize.sh_):

GMT_segmentize

DVC has been updated with the new and removed plots.

### CODE REVIEW

**Comment by PaulWessel** [comment]
No not goo, not sure what I did, pinging @seisman for assistance

**Comment by seisman** [comment]
Not sure what happened, maybe because @PaulWessel was working on several PRs that change DVC files at the same time.

Now this PR has conflicts, which must be resolved first.

**Comment by PaulWessel** [comment]
Conflicts fixed.

File: doc/scripts/images.dvc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 outs:
-- md5: e0c1334651a2b06141769144677fdff5.dir
-  size: 33377667
-  nfiles: 204
+- md5: 453d9a7563e7c2ec20cc0ed14716053c.dir
```
Comment (seisman) [inline doc/scripts/images.dvc:2]: The changes doesn't make sense to me. I expect to see `nfiles` in `doc/scripts/images.dvc` increase by 1, and `nfiles` in `test/baseline/test.dvc` decrease by 1.

Comment (Esteban82) [reply doc/scripts/images.dvc:2]: At the top, Paul said that he removed one test.

**Comment by PaulWessel** [comment]
Hi @maxrjones, any idea what is going on here? A few things I dont understand:

The PS files are not in our GitHub anymore, just DVC.  If I have a PR like this one that adds one new doc script and corresponding image but also removes a script, I have deleted that postscript file from test/baseline/module and used dvc add <directory> to update the .git file. Should I not/never delete files added to DVC?

**Comment by PaulWessel** [comment]
Ok, I did dvc add docs/scripts/images again and got a new .git file with 204 files.  Seems fine?

**Comment by seisman** [comment]
> Ok, I did dvc add docs/scripts/images again and got a new .git file with 204 files. Seems fine?

Currently, I can't see any changes in the `docs/scripts/images.dvc` file.

**Pull request merged**